### DISCUSSION TITLE
Unable to delete the existing folder

### DISCUSSION THREAD

**Discussion by KisetsuLee** [microsoft/vscode-cpptools#10114] [Q&A]
I run `rm -rf .vscode-server/extensions/ instead of ls .vscode-server/extensions` accidently.
when I try to install extension again in remote, the issue reported even though I'd deleted this folder manually.

Is this a bug? and what should I do to install it again?

> Unable to delete the existing folder '/home/user/.vscode-server/extensions/ms-vscode.cpptools-1.12.4-linux-x64' while installing the extension 'ms-vscode.cpptools'. Please delete the folder manually and try again

**Comment by sean-mcmanus** [comment]
Are you sure `/home/user/.vscode-server/extensions/ms-vscode.cpptools-1.12.4-linux-x64` is deleted and no processes from that folder are running? Restarting the OS might help. Otherwise, you might try asking at https://github.com/microsoft/vscode/issues , since our extension doesn't handle our own installation.

**Comment by KisetsuLee** [comment]
I reinstall pre-release version successfully. Then uninstall pre-release version, and reinstall 1.12.4 again, this time installation is ok~
